From 031abf80020b2fa75850d6e09f4489b687a5cb19 Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jirislaby@gmail.com>
Date: Sun, 24 Jun 2012 15:40:40 +0200
Subject: [PATCH] rename grub info file to grub2

Signed-off-by: Jiri Slaby <jirislaby@gmail.com>

From: Andrey Borzenkov <arvidjaar@gmail.com>
Do not rename file here. quilt does not support it and creates the
whole file if patch needs refreshing. It means that to regenerate two
files - Makefile.core.am and  Makefile.util.am - it may be necessary to
manually rename it.
---

--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -1,7 +1,7 @@
 AUTOMAKE_OPTIONS = subdir-objects
 
 # AM_MAKEINFOFLAGS = --no-split --no-validate
-info_TEXINFOS = grub.texi grub-dev.texi
+info_TEXINFOS = grub2.texi grub-dev.texi
 grub_TEXINFOS = fdl.texi
 
 EXTRA_DIST = font_char_metrics.png font_char_metrics.txt
--- a/docs/grub.texi
+++ b/docs/grub.texi
@@ -1,7 +1,7 @@
 \input texinfo
 @c -*-texinfo-*-
 @c %**start of header
-@setfilename grub.info
+@setfilename grub2.info
 @include version.texi
 @settitle GNU GRUB Manual @value{VERSION}
 @c Unify all our little indices for now.
